doc/src/manual/metaprogramming.md

```diff
@@ -1119,6 +1119,7 @@ have to know whether you're calling a regular or generated function. Let's see h
 ```jldoctest generated
 julia> x = foo(2); # note: output is from println() statement in the body
 Int64
+4
```

Comment:
Did you try this? This should not print due to the semicolon.

Reply:
Exactly! I had omitted the semicolon, magic.